### ISSUE TITLE
Server broken with npm 3

### ISSUE THREAD

**Issue by k-lange** [mklabs/tiny-lr#86]
After installing tiny-lr (well, actually [gulp-livereload](https://github.com/vohof/gulp-livereload)) I get the following error when I turn on the liverload extension in chrome:
 ```
Error: ENOENT, open '<path>/node_modules/tiny-lr/node_modules/livereload-js/dist/livereload.js'
  at Error (native)
```
This is probably because npm 3 is trying to install dependencies [maximally flat](https://github.com/npm/npm/releases/tag/v3.0.0) and thus [this line](https://github.com/mklabs/tiny-lr/blob/master/lib/server.js#L22) does not work anymore. I don't really know an elegant solution, but it sure would be nice to be able to use this module with npm 3.

**Comment by shama** [comment]
Fixed by 5754e8cc0ac2b430c78b265a84b459b054aaa889

**Comment by k-lange** [comment]
Sorry, just checked the issues and did not see anything. Thanks!

**Comment by redox** [comment]
I provided a reproduction use-case in https://github.com/mklabs/tiny-lr/issues/71#issuecomment-138914911

**Comment by OnkelTem** [comment]
Commit 5754e8cc0ac2b430c78b265a84b459b054aaa889 was not released in 0.1.6.
As a temporary solution just apply the fix manually.

@shama?
